src/components/landing-page.tsx

```
@@ -0,0 +1,837 @@
+						transition={{ duration: 0.8, delay: 0.2 }}
+						viewport={{ once: true }}
+					>
+						<p className="text-sm text-white/60">
+							© 2025 Vel. All rights reserved. Powered by Lisk Blockchain.
+						</p>
+						<div className="mt-4 flex gap-6 md:mt-0">
+							{["Privacy Policy", "Terms of Service", "Cookie Policy"].map(
+								(item, index) => (
+									<motion.a
+										key={index}
+										href="#"
```

Biome (1.9.4)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)